src/20240905_AaveV3Ethereum_OrbitProgramRenewalQ32024/OrbitProgramRenewalQ32024.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Orbit Program Renewal - Q3 2024"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-orbit-program-renewal-q3-2024/18834"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xea325542397fce607755f6c14be407f60a71a81f3a23c6b3a67e298b9dd8c091"
+---
+
+## Simple Summary
+
+Proposing the renewal of the Orbit program for recognized delegates, compensating them with GHO and ETH reimbursement of Gas costs , associated with their governance activity during part of Q3 2024 ( From 2024-05-13 to 2024-08-30).
```

Comment:
Missing explanation/mention of the overdue costs transfers 2.2k.
Same in specificaiton

Reply:
done